style/js/README.md

```diff
@@ -1,3 +1,163 @@
+JavaScript Style Guides
+=====
+
+Guides for programing JavaScript in style
+
+General
+---
+* Use [ESLint](http://eslint.org/)
+* Always use `use strict`
+
+Parens and Braces
+---
+* Leave one space between `if` and `(`.
+
+```javascript
+if (something) {
+  //some work here;
+}
+```
+
+* Prefer this form for multiline if statements.
+
+```javascript
+if (something) {
+  // do stuff
+  // even more stuff
+} else {
+  // some other stuff
+  // real stuff
+}
+```
+
+* Don't include spaces before `(` and function declaration.
+
+```javascript
+function bar() {
+  return  foo;
+}
+
+var bar = function() {
+  return foo;
+}
+```
+
+Assignments and Declarations
+---
+* Initialize arrays with `[]`.
+
+```javascript
+var array = [];
+```
+
+* Initialize objects with `{}`.
+
+```javascript
+var object = {};
+```
+
+* Leave one space after `{` and before `}` when initializing objects on the same line.
+
+```javascript
+var map = { ready: 9, when: 4, "you are": 15 };
+```
+
+* Prefer single line object/array initialization.
+
+```javascript
+var map = { when: 4, ready: 9, "you are": 15 };
+
+var map = [1, 2, 3];
+```
+
+Constructors
+---
+
+* Prefer this form for constructors.
+
+```javascript
+function Foo(bar) {
+  this.bar = bar;
+}
+
+Foo.prototype.getBar = function() {
```

Comment:
We're using blue (which is es6) in the blog (which is rails). There might have been some quirks at the time, regarding file extensions, but it worked perfectly fine otherwise